RELEASE.rst

```diff
@@ -4,7 +4,7 @@ This release introduces a :attr:`~hypothesis.settings.deadline`
 setting to Hypothesis.
 
 When set this turns slow tests into errors. By default it is unset but will
-warn if you exceed 500ms, which will become the default value in a future
+warn if you exceed 300ms, which will become the default value in a future
```

Comment:
300 here but 200 in implementation?

Reply:
Oops. It helps to save the file you're editing. I originally went for 300, then decided that actually I preferred 200 - if gives you a comfortable amount of time to get in under the timeout with other things that are not being counted contributing time.